src/Button/Button.less

```diff
@@ -1,34 +1,50 @@
-@import "../less/colors.less";
 @import "../less/fonts.less";
+@import (reference) "../less/colors.less";
+@import (reference) "../less/sizing.less";
+@import (reference) "../less/type-size.less";
+
+
+.buttonColor(@backgroundColor: @primary_blue) {
+  background-color: @backgroundColor;
+  border-color: @backgroundColor;
+  .shade(border-bottom-color, @backgroundColor, 2);
+
+  &:not([disabled]) {
+    &:hover,
+    &:focus,
+    &:active {
+      .shade(background-color, @backgroundColor, 2);
+      .shade(border-color, @backgroundColor, 2);
+      .shade(border-bottom-color, @backgroundColor, 4);
+    }
+  }
+}
 
 a, button {
   &.Button {
-    background: #ffffff;
-    border: 0;
-    border-top-right-radius: 3px;
-    border-top-left-radius: 3px;
+    .type_medium;
+    border-radius: 3px 3px 0 0;
+    border: 1px solid transparent;
+    border-bottom-width: 3px;
+    box-sizing: border-box;
```

Comment:
not too much of a difference, but sizing ends up being a little more proportional to the text size.
shading on hover is also a little darker/more noticeable.

added screenshots to the PR description